lib/Factory/UserFactory.php

```diff
@@ -189,14 +194,11 @@ public function getDoohUsers()
     /**
      * Get system user
      * @return User
+     * @throws NotFoundException
      */
     public function getSystemUser()
     {
-        $user = $this->create();
-        $user->userId = 1;
-        $user->userName = 'system';
-        $user->userTypeId = 1;
-        $user->email = $this->configService->getSetting('mail_to');
+        $user = $this->getById($this->configService->getSetting('SYSTEM_USER'));
```

Comment:
Do we still need to override the email address here? Or have we traced through where we use the system user for emailing, and replaced with the global `mail_to` at that point?